hypha/apply/review/templates/review/review_edit_form.html

```diff
@@ -10,37 +10,49 @@
+                {{ form.media }}
+                {% csrf_token %}
 
-    <div class="wrapper wrapper--medium wrapper--inner-space-medium">
-        <form class="form form--with-p-tags form--scoreable" action="" method="post" novalidate>
-            {{ form.media }}
-            {% csrf_token %}
+                {% for hidden in form.hidden_fields %}
+                    {{ hidden }}
+                {% endfor %}
 
-            {% for hidden in form.hidden_fields %}
-                {{ hidden }}
-            {% endfor %}
+                {% for field in form.visible_fields %}
+                    {# to be replaced with better logic when we use stream form #}
+                    {% ifchanged field.field.group %}
+                        {% for key, value in form.titles.items %}
+                            {% if key == field.field.group %}
+                                <h2>{{ value }}</h2>
+                            {% endif %}
+                        {% endfor %}
+                    {% endifchanged %}
 
-            {% for field in form.visible_fields %}
-        {# to be replaced with better logic when we use stream form #}
-                {% ifchanged field.field.group %}
-                    {% for key, value in form.titles.items %}
-                        {% if key == field.field.group %}
-                            <h2>{{ value }}</h2>
-                        {% endif %}
-                    {% endfor %}
-                {% endifchanged %}
-
-                {% if field.field %}
-                    {% include "forms/includes/field.html" %}
-                {% else %}
-                    {{ field }}
+                    {% if field.field %}
+                        {% include "forms/includes/field.html" %}
+                    {% else %}
+                        {{ field }}
+                    {% endif %}
+                {% endfor %}
+                {% if not object.id or object.is_draft %}
+                    <button class="button button--submit button--top-space button--white" type="submit" name="{{ form.draft_button_name }}">{% trans "Save draft" %}</button>
                 {% endif %}
-            {% endfor %}
-            {% if not object.id or object.is_draft %}
-                <button class="button button--submit button--top-space button--white" type="submit" name="{{ form.draft_button_name }}">{% trans "Save draft" %}</button>
-            {% endif %}
-            <button class="button button--submit button--top-space button--primary" type="submit" name="submit">{% trans "Submit" %}</button>
-        </form>
+                <button class="button button--submit button--top-space button--primary" type="submit" name="submit">{% trans "Submit" %}</button>
+            </form>
+        </section>
+        {% if show_running_total %}
+            <div class="form--score-box inline-block px-4 py-2 mt-1 text-center sticky top-1" hidden>
+                {% trans "Score:" %} <span class="score-number pe-1">0</span>
```

Comment:
```suggestion
                {% trans "Score:" %} <span class="score-number">0</span>
```

padding left is not needed now.